components/brave_rewards/resources/shared/components/wallet_card/rewards_summary.style.ts

```diff
@@ -102,6 +101,10 @@ export const dataTable = styled.div`
 
   tr:last-child {
     border-bottom: none;
+
+    td:last-child {
+      padding-bottom: 16px;
+    }
```

Comment:
@zenparsing This was the cleanest approach I could find, but I'm still not too enthusiastic about it. Padding a table row is not supported in any kind of clean way in CSS that I could find. Another approach I toyed with was having a transparent border-bottom that acted like padding. I felt like this approach was slightly more readable. Open to other ideas, though!

Reply:
Seems fine to me!